### PR TITLE
new: support search placeholder

### PR DESCRIPTION
**Pull request by TTWShell**

### CODE REVIEW

**Comment by uSpike** [comment]
This wipes out translations for the search placeholder field.  Could you use `flask_admin.babel.gettext` to get the "search" string?

**Comment by TTWShell** [comment]
@uSpike

File: flask_admin/contrib/sqla/view.py [hunk 1]
```diff
@@ -555,6 +555,33 @@ def init_search(self):
...
+
+        for searchable in self.column_searchable_list:
+            if isinstance(searchable, InstrumentedAttribute):
+                placeholders.append(
+                    self.column_labels.get(searchable.key, searchable.key))
+            else:
+                placeholders.append(
+                    self.column_labels.get(searchable, searchable))
+
+        return 'Search: %s' % u', '.join(placeholders)
```
Comment (illume) [inline flask_admin/contrib/sqla/view.py]: Does gettext need to be used here?

**Comment by TTWShell** [comment]
@pawl How about this pr?

File: flask_admin/contrib/sqla/view.py (hunk 1 above)
Comment (xqliu) [inline flask_admin/contrib/sqla/view.py]: @illume Do you have any insight here on what will happen if we use lazy_gettext in column_labels, and then use self.column_labels.get, will it get the already translated label?

gettext might not necessary here.

**Comment by petrus-jvrensburg** [comment]
This looks good to me. Any objections to merging?

One change that I would make would be to call `_gettext` explicitly on the substring `Search` from the template, to ensure it gets translated appropriately. I don't see the use in calling `gettext` on the list of column names.

**Pull request merged**